Description: Fix slim template loading wrt load order
 If Slim has been loaded before that point, an instance variable won't be set.
Author: Antonio Terceiro <terceiro@debian.org>
Origin: vendor
Forwarded: https://github.com/asciidoctor/asciidoctor/pull/1576
Last-Update: 2015-11-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/asciidoctor/converter/template.rb
+++ b/lib/asciidoctor/converter/template.rb
@@ -253,10 +253,10 @@ module Asciidoctor
           unless defined? ::Slim
             # slim doesn't get loaded by Tilt, so we have to load it explicitly
             Helpers.require_library 'slim'
-            if @safe && ::Slim::VERSION >= '3.0'
-              slim_asciidoc_opts = (@engine_options[:slim][:asciidoc] ||= {})
-              slim_asciidoc_opts[:safe] ||= @safe
-            end
+          end
+          if @safe && ::Slim::VERSION >= '3.0'
+            slim_asciidoc_opts = (@engine_options[:slim][:asciidoc] ||= {})
+            slim_asciidoc_opts[:safe] ||= @safe
           end
           # load include plugin when using Slim >= 2.1
           require 'slim/include' unless (defined? ::Slim::Include) || ::Slim::VERSION < '2.1'
